### ISSUE TITLE
Upgrade to HA version 2024.7.0 complains about issue with Visonic integration waiting on "async_forward_entry_setup"

### ISSUE THREAD

**Issue by haforme** [davesmeghead/visonic#117]
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:151
First occurred: 9:12:40 PM (1 occurrences)
Last logged: 9:12:40 PM

Detected code that calls async_forward_entry_setup for integration visonic with title: Panel 0 (PowerMax+) and entry_id: 0aeac5549fc94e78803ea11516b913cd, during setup without awaiting async_forward_entry_setup, which can cause the setup lock to be released before the setup is done. This will stop working in Home Assistant 2025.1. Please report this issue.

**Comment by davesmeghead** [comment]
Hi,
Thanks but I already know about it, it's posted on the developer forums here
https://developers.home-assistant.io/blog/2024/06/12/async_forward_entry_setups

Although it does say on the blog that it will keep working until 2025.6 release i.e. June next year.
I've already fixed this in the dev release on Github, I just haven't put it in the Master release yet.
https://github.com/davesmeghead/visonic/tree/dev

I'll leave this comment open so others see this and don't create a new issue.

**Comment by davesmeghead** [comment]
I've pulled the Dev to the Master so I'll close this Issue
